src/middlewares/analyticsMiddleware/index.js

```diff
@@ -0,0 +1,19 @@
+import { newWalletEvents } from './newWalletEvents'
+import { sendTransactionEvents } from './sendTransactionEvents'
+import { userParams } from './userParams'
+
+export const analyticsMiddleware = store => next => (action) => {
+  try {
+    const startState = store.getState()
+    newWalletEvents(startState, action)
+    sendTransactionEvents(startState, action)
+  } catch (err) {
```

Comment:
would be better to split events by our routes (as well as reducers) "domains",
like `walletsEvents`, `digitalAssetsEvents` (word `Events` better to skip also)